syntax_and_semantics/requiring_files.md

```diff
@@ -33,6 +33,8 @@ The second rule is very convenient because of the typical directory structure of
     - project_spec.cr
 ```
 
+For example, if you want to `require "foo"` in "project.cr", then you should run crystal in the `project` directory with `crystal src/project.cr`. This ensures that the `lib` folder is one of the require paths.
```

Comment:
This sounds to bulky, and it should better describe *how* it behaves if used as intended and how you break it.
```
For example, if you put `require "foo"` in `project.cr` and run `crystal src/project.cr` in the
project's root directory, it will find `foo` in `lib/foo/foo.cr`.
If you run the compiler from somewhere else, say the `src` folder,
`lib` will not be in the path and `require "foo"` can't be resolved.
```

Reply:
I agree with your suggestion and will update my pull request with the new text. Hope everything is better now!